frontend/components/common/formatters/AnimatedNumber.vue

```diff
@@ -32,8 +32,17 @@ export default Vue.extend({
             type: Number,
             default: undefined,
         },
+        disableThousandSeparators: {
+            type: Boolean,
+            default: false,
+        },
     },
     computed: {
+        formattedValue(): string {
```

Comment:
We need this formattedValue computed property, as we need to pass a certain type of number to the external `AnimatedNumber` component. 

It takes in either a string or a number, which is why I decided to use our `formatToAutomaticDecimalPoints` function twice, as it ensures that we do not lose any kind of precision when converting the BigNumber to a string or number.